examples/strong/lib/common_fixes_analysis.dart

```diff
@@ -82,13 +82,13 @@ class HoneyBadger extends Animal {
 //-----------------------------------------------
 
 // #docregion func-T
-typedef bool Filter<T>(T any);
+typedef Filter<T> = bool Function(T any);
 Filter<String> filter = (String x) => x.contains('Hello');
 // #enddocregion func-T
 
 //-----------------------------------------------
 
 // #docregion func-cast
-typedef bool Filter1(dynamic any);
+typedef Filter1<T> = bool Function(T any);
```

Comment:
This should be:

```dart
typedef Filter1 = bool Function(dynamic any);
```

Shouldn't it?

Reply:
Yes (they should be equivalent, but I agree that the original explicit use of `dynamic` there is better).

Fixed.